src/guides/reset_with_sd_card.html

```diff
@@ -0,0 +1,61 @@
+---
+layout: troubleshooting-guide
+description: Home Assistant Green
+guideName: Resetting the device using an SD card
+tools:
+  title: Prerequisites
+  image: green_sd_card.png
+  content: |
+    <ul>
+        <li>Home&nbsp;Assistant Green</li>
+        <li>microSD card</li>
+        <li>Home Assistant 13 or older and you cannot upgrade to Home Assistant 13.1 or later<ul><li>To see which version you have, go to <a class="my" href="https://my.home-assistant.io/redirect/info/" target="_blank"><b>Settings</b> > <b>About</b></a>.</li></ul></li>
```

Comment:
In this guide I would simply drop this. There is no version prerequisite, this works for any version.

```suggestion
        <li>Home Assistant 13 or older and you cannot upgrade to Home Assistant 13.1 or later<ul><li>To see which version you have, go to <a class="my" href="https://my.home-assistant.io/redirect/info/" target="_blank"><b>Settings</b> > <b>About</b></a>.</li></ul></li>
```